fashrevwall/test_twitter_client.py

```diff
@@ -0,0 +1,16 @@
+import pytest
```

Comment:
Oops, I started the tests, but never finished them :) It was useful to get the client started and demonstrate how to use it, but I can get rid of them now. Do you want me to get rid of this and run_streaming.py?